cards_corrections.yml

```diff
@@ -1,4 +1,36 @@
 # Objects to be fixed
+#
+# Format is
+#
+# Card Name:
+#   field to fix: new value
+#
+# Ex:
+#
+# Jace, the Planeswalker:
+#   loyalty: 5
+#   manaCost: 1UUB
+#
+# Fields and their values
+# name: Card Name
+# cmc: 4 (int)
+# colorIdentity:
+#   - U
+#   - B
+# colors:
+#   - Blue
+#   - Black
+# manaCost: 1UUB
+# number: 140
+# rarity: Mythic Rare
+# power: X
+# text: "{5}, {T}: You win the game."
+# toughness: *
+# type: Legendary Artifact Creature - Human Monk
+# url: http://wizards.c0m/link/to/card.png
```

Comment:
I agree with that _but_ what if a source says it's a 4/3 and it's actually a 4/4? We are trying to design it so once they fix we bust our cache and get the update, but if that was a few days, we'd probably want to allow a spot correction via the file. 

Ergo, we should _advertise_ that those types of problems should be fixed in code, and we should fix them in code if possible, but also design the software so it would allow a correction of that type.